kong/plugins/jwt/handler.lua

```diff
@@ -189,6 +189,12 @@ local function do_authentication(conf)
 
   set_consumer(consumer, jwt_secret, token)
 
+  for _, claim_name in ipairs(conf.inject_claims) do
```

Comment:
Better wrap this loop within a branch checking for the existence of `conf.inject_claims`, we've had related issues in the past:
```lua
if conf.inject_claims then

end
```